manifests/server.pp

```diff
@@ -158,18 +159,43 @@
           }
         }
       }
+    }
 
-      # Wrap the replset class
-      class { 'mongodb::replset':
-        sets => $replset_config_REAL
-      }
-      Anchor['mongodb::server::end'] -> Class['mongodb::replset']
+    # Wrap the replset class
+    class { 'mongodb::replset':
+      sets => $replset_config_REAL
```

Comment:
See line 136/137 of `if $replset {`

If $replset is undef, previously `Class['mongodb::replset']` would not have been declared, but now it will be. And when it's declared `$replset_config_REAL` will not be defined (because it is only defined if `$replset` is not undef).

Is this intended?